nimbus-core/src/main/java/com/antheminc/oss/nimbus/app/extension/config/DefaultProcessConfig.java

```diff
@@ -125,10 +126,16 @@ public CommandExecutorTaskDelegate commandExecutorTaskDelegate(BeanResolverStrat
 	public FunctionHandler<?, ?> queryFunctionHandler(){
 		return new DefaultSearchFunctionHandlerQuery<>();
 	}
-
+	
 	@Bean(name="default._process$execute?fn=_eval")
 	public EvalFunctionHandler<?,?> evalFunctionHandler(ExpressionManager expressionManager){
 		return new EvalFunctionHandler(expressionManager);
 	}
 
+	@Bean(name = "multipartResolver")
```

Comment:
Avoid adding to scattered bean configs.

Reply:
@sandeepmantha can you look at this one?

Reply:
We will bring back the `FileImporterGateway` and let the framework clients create this bean themselves.

Reply:
Fixed.